doc/sphinx/source/projects/index.rst

```diff
@@ -5,7 +5,7 @@ Firmware Applications
    :maxdepth: 2
    :caption: Contents:
 
-   /source/projects/ad4130_iio/index.rst
+   /source/projects/ad4130_iio/ad4130_iio.rst
```

Comment:
Any reason this change has been made only to the AD4130?

Reply:
The plan is to remove the intermediate contents page for all the projects as it feels redundant. Wanted to know your thoughts on this.